proxy/http2/Http2ClientSession.cc

```diff
@@ -183,7 +181,10 @@ Http2ClientSession::start()
 
   this->connection_state.init();
   send_connection_event(&this->connection_state, HTTP2_SESSION_EVENT_INIT, this);
-  this->handleEvent(VC_EVENT_READ_READY, read_vio);
+
+  if (this->sm_reader->read_avail() > 0) {
```

Comment:
is_read_avail_more_than?

Reply:
Should be. I missed pushed latest one using it.